editor/components/rich-text/index.js

```diff
@@ -695,20 +695,9 @@ export class RichText extends Component {
 		}
 	}
 
-	updateContent() {
-		// Do not trigger a change event coming from the TinyMCE undo manager.
-		// Our global state is already up-to-date.
-		this.editor.undoManager.ignore( () => {
-			const bookmark = this.editor.selection.getBookmark( 2, true );
-
-			this.savedContent = this.props.value;
-			this.setContent( this.savedContent );
-			this.editor.selection.moveToBookmark( bookmark );
```

Comment:
- Dropping the focus management breaks the "merge". The focus is not restored at the right position when merging two paragraphs. 

 - I believe the undo behavior is also necessary in this case or in cases the `value` of a RichText is updated programmatically (the change is not triggered by the RichText element itself)

Reply:
But all tests have passed successfully!

😉

Reply:
hahaha :-) we definitely need a test for that

Reply:
Added a failing test in dd0e347.